include/seqan3/version.hpp

```diff
@@ -28,6 +28,24 @@
                      + SEQAN3_VERSION_MINOR * 100 \
                      + SEQAN3_VERSION_PATCH)
 
+/*!\brief Converts a number to a string.
+ * Preprocessor needs this indirection to
```

Comment:
I think, you can put these on two lines, instead of three.

Reply:
Yes, I should do that and will! :-)